src/runtime/tests/manifest-test.ts

```diff
@@ -4879,57 +4899,6 @@ recipe
       assert.isTrue(manifest.recipes[0].normalize());
     });
   });
-  it('parses schema field annotations', async () => {
-    const manifestStr = `
-annotation hello(txt: Text)
-  targets: [SchemaField]
-  retention: Source
-  doc: 'hello world'
-schema Foo
-  field1: Text @hello @ttl(value: '3d')
-    `;
-    const manifest = await Manifest.parse(manifestStr);
-    const fooSchema = manifest.findSchemaByName('Foo');
-    assert.lengthOf(Object.keys(fooSchema.fields), 1);
-    const annotations = fooSchema.fields['field1'].annotations;
-    assert.lengthOf(annotations, 2);
-    assert.isNotNull(annotations.find(a => a.name === 'hello'));
-    assert.equal(annotations.find(a => a.name === 'ttl').params['value'], '3d');
-    assert.equal(manifest.toString(), manifestStr.trim());
-  });
-  it('parses inline schema field annotations', async () => {
```

Comment:
i don't think this one is a duplicate - it declares an annotation on a schema field of the inline schema in a particle handle connection spec.

Reply:
there is another test called "parses inline schema field annotations" at line 4828 (in this version) as it is identical to this one

Reply:
yes, see that now, sorry for the confusion, thanks for cleaning this.